persistence-modules/r2dbc/src/main/java/com/baeldung/examples/r2dbc/flyway/config/DatabaseConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.examples.r2dbc.flyway.config;
+
+import org.flywaydb.core.Flyway;
+import org.springframework.boot.autoconfigure.flyway.FlywayProperties;
+import org.springframework.boot.autoconfigure.r2dbc.R2dbcAutoConfiguration;
+import org.springframework.boot.autoconfigure.r2dbc.R2dbcProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@EnableConfigurationProperties({ R2dbcProperties.class, FlywayProperties.class })
```

Comment:
The purpose for having this annotation here is so that we are asking Spring to ensure that there is a bean created for these 2 properties classes, since they are used to create the `Flyway` bean